rest-services/it.tests/src/test/java/com/_4point/aem/docservices/rest_services/it_tests/client/forms/ExportDataTest.java

```diff
@@ -61,20 +81,30 @@ void setUp() throws Exception {
 	@Test
 	@DisplayName("Test exportData() Happy Path.")
 	void testExportData() throws Exception {
-
+	
 		Document pdforxdp = SimpleDocumentFactoryImpl.INSTANCE.create(SAMPLE_FORM_PDF.toFile());
 		DataFormat dataformat = com.adobe.fd.forms.api.DataFormat.XmlData;
 		Document pdfResult = underTest.exportData(pdforxdp, dataformat);
-		byte[] data= pdfResult.getInlineData();
-         org.w3c.dom.Document document = DocumentBuilderFactory.newInstance().newDocumentBuilder().parse(new
-         ByteArrayInputStream(pdfResult.getInlineData()));
-          DOMSource domSource = new DOMSource(document);
-         String aa=domSource.toString();
-         
-   assertThat("Expected a xml to be returned.", ByteArrayString.toString(aa.getBytes(),8), containsString("%, X, M, L, -, 1, ., 7"));
-
+		
+		 org.w3c.dom.Document document = DocumentBuilderFactory.newInstance().newDocumentBuilder().parse(new ByteArrayInputStream(pdfResult.getInlineData())); 
+		 XPathExpression xpath = XPathFactory.newInstance().newXPath().compile("/form1");
+		 Node output1 = (Node)xpath.evaluate(document, XPathConstants.NODE);
+		 output1.normalize();
+		 String content = output1.getTextContent();
+		 output1.normalize();
+		 
+		 System.out.println(content.contains("abcd"));
+		 
+		 assertEquals("form1",output1.getNodeName());
+		 
+		System.out.println(content.getBytes());
```

Comment:
Can we remove this?  In general you want to keep your tests to be quiet unless there is a failure.  This makes it easier to spot when a test is failing.  You're not going to miss a problem because it is in the middle of background noise.

If what is being printed is important for pass/failure of a test, then it should be contained in an assertion that passes or fails.  If it is information that is handy/important to know when debugging issues, then just comment it out so that someone can uncomment it later when they are debugging.  If it is just code of little significance left over from previous iterations of the code, then just remove it.